packages/cli/src/plugins/etherscan.ts

```diff
@@ -32,7 +32,8 @@ const apiUrls = {
   [42220]: 'https://api.celoscan.io/api',
   [44787]: 'https://api-alfajores.celoscan.io/api',
 }
-type ChainId = keyof typeof apiUrls
+
+export type ChainId = keyof typeof apiUrls
```

Comment:
Might need to export this from `exports/plugins.ts`. Maybe rename to `EtherscanChainId` to avoid future name clashes.

```suggestion
export type EtherscanChainId = keyof typeof apiUrls
```

Reply:
done @jxom